db/migrate/20220915125503_create_comments.rb

```diff
@@ -0,0 +1,12 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.integer :authorId
+      t.integer :postId
+      t.text :text
+
+
+      t.timestamps
+    end
+  end
+end
```

Comment:
- Kindly make sure Foreign key is included with the corresponding index 👍🏽 
for example should have something like
```ruby
create_table :comments do |t|
    t.text :text
    t.timestamps 
    t.references :author, foreign_key: { to_table: 'users' } # this will create author_id and a corresponding index
    t.references :post, foreign_key: true # this will create post_id and a corresponding index 
  end
```